src/components/flyout/flyout_body.styles.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { css } from '@emotion/react';
+import { UseEuiTheme } from '../../services';
+import {
+  logicalCSS,
+  logicalCSSWithFallback,
+  euiYScrollWithShadows,
+} from '../../global_styling';
+
+export const euiFlyoutBodyStyles = (euiThemeContext: UseEuiTheme) => {
+  return {
+    euiFlyoutBody: css`
+      ${logicalCSSWithFallback('overflow-y', 'hidden')}
+      ${logicalCSS('height', '100%')}
+    `,
+    overflow: {
+      euiFlyoutBody__overflow: css`
+        ${euiYScrollWithShadows(euiThemeContext)};
+      `,
+      hasBanner: css`
+        ${euiYScrollWithShadows(euiThemeContext, {}, 'end')};
+      `,
+    },
+    euiFlyoutBody__banner: css`
```

Comment:
If this CSS/class is getting applie directly to `euiFlyoutBody`, this should be just `hasBanner` and not `__banner` (since the CSS isn't actually getting applied to `.euiFlyoutBody__banner`
```suggestion
    hasBanner: css`
```